### ISSUE TITLE
Postgrest

### ISSUE THREAD

**Issue by evanzummeren** [HDAT/hdat-backend#1]
Ken je dit project? Best wel nice

http://www.postgrest.com

`brew install postgrest`

Vervolgens

`postgrest --db-host localhost --db-port 5432 --db-name bgb --db-pool 200 --anonymous $USER --port 3000 --db-user $USER --v1schema public`

Daarna:
localhost:3000/bgbPlace?select=naam,regio,id&id=gte.900

**Comment by Zaturrby** [comment]
Nee. maar het klink idd als iets wat we nodig hebben straks om van al die loss JSON’s af te komen. Apiary is ook zoiets. Probleem is vooral hosting waar ik destijds tegenop zag; maar dat heb ik nu ook onder de knie met digitalocean, kost wel een 10tje per maand, maar is verder eindeloos.
